Get size

NeutryFD opened the pull request:
Show size of the  disk in the device_info metric table

<img width="655" height="252" alt="disk_size_metric" src="https://github.com/user-attachments/assets/96a47ee0-7f6a-4d0a-8a91-1b220ad59d0c" />

NeutryFD committed:
add size of disk into device_info metric

Commit:
add new disk_size metric to the dashboard

micha37-martins commented:
Hi and thanks for the PR! I tested it and found out that it does not work for me. The .prom file causes error with the dashboard you provided on my system. Feel free to update your PR and also bump the Version in `smartmon.sh` (currently smartmon_version="0.3.16" )

NeutryFD committed:
Fix undefined model_name variable and update version to 0.3.17

- Changed disk_total_bytes metric to user_capacity_bytes
- Removed undefined model_name variable reference from metric labels
- Bumped version from 0.3.16 to 0.3.17
- Fixes dashboard error reported in PR review

NeutryFD committed:
Fixes dashboard compatibility with updated metric output

Commit:
Fix Prometheus label compatibility and update dashboard to v11.6.0
